File: java2csharp/java/Samsung/ProfessionalAudio.FloatingConotrller.WidgetsLibrary/IFloatingControlerHandler.java
package com.samsung.android.sdk.professionalaudio.widgets;

import java.util.Map;

import android.view.ViewGroup;

import com.samsung.android.sdk.professionalaudio.app.SapaAppService;

interface IFloatingControlerHandler {

    int getLayoutResource();

    void initLayout(ViewGroup layout);

    void setSapaAppService(SapaAppService sapaAppService, String mainPackagename);

    void stopConnections();

    boolean getBarExpanded();

    public Map<String, Boolean> getDevicesExpanded();

    int getBarAlignment();

    void loadBarState(FloatingController floatingController, int barAlignment);
}

File: java2csharp/csharp/Samsung/ProfessionalAudio.FloatingConotrller.WidgetsLibrary/IFloatingControlerHandler.cs
using System.Collections.Generic;

namespace com.samsung.android.sdk.professionalaudio.widgets
{

	using ViewGroup = android.view.ViewGroup;

	using SapaAppService = com.samsung.android.sdk.professionalaudio.app.SapaAppService;

	internal interface IFloatingControlerHandler
	{

		int LayoutResource {get;}

		void initLayout(ViewGroup layout);

		void setSapaAppService(SapaAppService sapaAppService, string mainPackagename);

		void stopConnections();

		bool BarExpanded {get;}

		IDictionary<string, bool?> DevicesExpanded {get;}

		int BarAlignment {get;}

		void loadBarState(FloatingController floatingController, int barAlignment);
	}

}